compiler/util/files.cpp

```diff
@@ -312,7 +312,8 @@ void closeInputFile(FILE* infile) {
 }
 
 
-static const char** inputFilenames = {NULL};
+static const char** inputFilenames = NULL;
+static int numInputFiles = 0;
```

Comment:
You could make `numInputFiles` a static variable in `testInputFiles`. It shouldn't change anything except moving the definition into the only function that uses it.

The only reason I see for it to be out here is if you add a check in `nthFilename(int i)` that `i < numInputFiles`